projects/plugins/jetpack/class.jetpack-admin.php

```diff
@@ -119,6 +122,73 @@ public function akismet_logo_replacement_styles() {
 		wp_add_inline_style( 'admin-bar', $style );
 	}
 
+	/**
+	 * Handle our Additional CSS menu item and legacy page declaration.
+	 *
+	 * This method was originally located in custom-css-4.7.php until $$next-version$$
+	 */
+	public static function additional_css_menu() {
+		// Add in our legacy page to support old bookmarks and such.
+		add_submenu_page( null, __( 'CSS', 'jetpack' ), __( 'Additional CSS', 'jetpack' ), 'edit_theme_options', 'editcss', array( __CLASS__, 'customizer_redirect' ) );
+
+		// Add in our new page slug that will redirect to the customizer.
+		$hook = add_theme_page( __( 'CSS', 'jetpack' ), __( 'Additional CSS', 'jetpack' ), 'edit_theme_options', 'editcss-customizer-redirect', array( __CLASS__, 'customizer_redirect' ) );
+		add_action( "load-{$hook}", array( __CLASS__, 'customizer_redirect' ) );
+
+	}
+
+	/**
+	 * Handle the redirect for the customizer.  This is necessary because
+	 * we can't directly add customizer links to the admin menu.
+	 *
+	 * This method was originally located in custom-css-4.7.php until $$next-version$$
```

Comment:
I'm guessing the `$$next-version$$` won't work here, based on the recognized patterns[ added here ](https://github.com/Automattic/jetpack/pull/20386/files)(if that's so, I wonder if they could be broader to include cases like this)?

Reply:
It would probably work out, but I'm still a little wary of a context-free replacement.

Reply:
As a workaround, would this work?

```
* This method was originally located in custom-css-4.7.php, and has been here @since $$next-version$$ .
```

Or perhaps even better:

```
* @since $$next-version$$ . Prior to that, this function was located in custom-css-4.7.php.
```

Reply:
I like the second option or something like it.